introduction.md

```diff
@@ -5,4 +5,10 @@ I like the alias D3V4N5H as it is truly Alpha-Numeric
 Another handle I use for personal branding is AdevNt. It is a play on the word "AdveNt".
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?